include/myst/clock.h

```diff
@@ -13,7 +13,7 @@ struct clock_ctrl
     long monotime0;
     volatile long now;
     unsigned long interval;
-    volatile int done;
+    volatile long done;
```

Comment:
Recommend adding the following compile-time checks
- Ensure the size of `clock_ctrl` is multiple of 8
- Ensure the offset of `now` is 8 byte aligned.

After this, adding comments in `_get_monotime` (where `now` is read) to emphasize that it's address is 8-byte aligned given that `ctr` is 8-byte aligned.